### PR TITLE
add Authorization data models

### PR DESCRIPTION
**Pull request by vraybaud**

### CODE REVIEW

File: authentication/jsonld-contexts/authentication.jsonld [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (bobeal) [inline authentication/jsonld-contexts/authentication.jsonld:1]: `authentication.jsonld` is not the more appropriate name for this context (actually, it deals with authorization and not with authentication).

so ... `authorization.jsonld` or a more extensible `security.jsonld`?

File: authentication/jsonld-contexts/authentication.jsonld [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	"@context": {
+		"name": "https://ontology.eglobalmark.com/authentication#name",
+		"username": "https://ontology.eglobalmark.com/authentication#username",
+		"role": "https://ontology.eglobalmark.com/authentication#role",
```
Comment (bobeal) [inline authentication/jsonld-contexts/authentication.jsonld:5]: or `roles` as it is not meant to be unique?

File: authentication/jsonld-contexts/authentication.jsonld (hunk 1 above)
Comment (vraybaud) [reply authentication/jsonld-contexts/authentication.jsonld:1]: authorization sounds nice to me

File: authentication/jsonld-contexts/authentication.jsonld (hunk 2 above)
Comment (vraybaud) [reply authentication/jsonld-contexts/authentication.jsonld:5]:
```
"roles": {
  "type": "Property",
  "value": ["admin","creator"]
}
```

is this how you see it?

File: authorization/jsonld-contexts/authorization.jsonld [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+	"@context": {
+		"name": "https://ontology.eglobalmark.com/authentication#name",
```
Comment (bobeal) [inline authorization/jsonld-contexts/authorization.jsonld:3]: you should also adapt the namespace to be consistent (i.e. change the `authentication` in it)

File: authorization/ngsild-payloads/Group.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      }
+   ],
+   "rCanAdmin": [
+      {
+         "type": "Relationship",
+         "object": "urn:ngsi-ld:Beehive:Beehive03"
+      }
+   ],
+   "@context": [
+      "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/authentication/jsonld-contexts/authentication.jsonld",
```
Comment (bobeal) [inline authorization/ngsild-payloads/Group.json:31]: This link will not be resolved

File: authorization/ngsild-payloads/Group.json [hunk 5]
```diff
@@ -0,0 +1,34 @@
+{
+   "id":"urn:ngsi-ld:Group:5acb5d63-fd74-4ad0-af5e-dc1fc1d4bc29",
+   "type":"Group",
```
Comment (HoucemKacem) [inline authorization/ngsild-payloads/Group.json:3]: What about the expansion/reduction of the Goup/User types ?
With the default context ?

Comment (vraybaud) [reply authorization/ngsild-payloads/Group.json:3]: done

**Pull request merged**